client/v2/client.go

```diff
@@ -80,6 +80,9 @@ type Client interface {
 	// Write takes a BatchPoints object and writes all Points to InfluxDB.
 	Write(bp BatchPoints) error
```

Comment:
Would there be a huge impact if we deleted this variant and forced all callers to use `WriteCtx`?

Reply:
I don't think a big impact but I didn't want to break the API when it is so easy to keep it unbroken.